src/components/organisms/NewFeed/index.js

```diff
@@ -119,20 +126,28 @@ function NewFeed({ onClickModalClose }) {
 
     if (inputAddress) {
       getCoordinates();
+      setAddress(inputAddress.label);
     }
   }, [inputAddress]);
 
   useEffect(() => {
-    if (!photoAddress && !inputAddress) {
-      setAddress("Please enter your address");
+    if (location[1] === 0) {
+      if (!photoAddress && !inputAddress) {
+        setAddress("Please enter your address");
```

Comment:
```suggestion
    if (location[1] === 0 && !photoAddress && !inputAddress) {
      setAddress("Please enter your address");
```
개인적인 의견이지만 이렇게 할 수도 있을 것 같습니다.🙂

Reply:
제안 감사합니다 수정하겠습니다~!